x11-packages/xwayland/xorg-server-1.20.1_os_utils.c.patch

```diff
@@ -52,21 +38,10 @@ diff -uNr xorg-server-1.20.1/os/utils.c xorg-server-1.20.1.mod/os/utils.c
          _exit(127);
      }
  
-@@ -1531,10 +1523,6 @@
-         free(cur);
-         return NULL;
-     case 0:                    /* child */
--        if (setgid(getgid()) == -1)
--            _exit(127);
--        if (setuid(getuid()) == -1)
--            _exit(127);
-         if (*type == 'r') {
-             if (pdes[1] != 1) {
-                 /* stdout */
-@@ -1577,20 +1565,7 @@
- 
-     return iop;
- #else
```

Comment:
The same applies here.

Reply:
Ok, sorry. Now I see it. It was like multiple referencing of setgid/setuid which was removed in later version of xwayland. Sorry for inconvenience.